### ISSUE TITLE
Removing fields via _config.yml using hidden_fields for Member->getMemberFormFields() gives errors

### ISSUE THREAD

**Issue by nimeso** [silverstripe/silverstripe-framework#4132]
The CMS/Security area returns errors when trying to view a Member
It seems that this ONLY happens when I try to remove LastVisited via _config.yml eg:

Member:
  hidden_fields:
    - 'LastVisited'

All the others seem fine eg:

Member:
  hidden_fields:
    - 'Image'
    - 'TimeFormat'
    - 'DateFormat'
    - 'Locale'
    - 'LockedOutUntil'
    - 'PasswordExpiry'
    - 'Salt'
    - 'PasswordEncryption'
    - 'AutoLoginExpired'
    - 'AutoLoginHash'
    - 'NumVisit'
    - 'RememberLoginToken'
    - 'TempIDExpired'
    - 'TempIDHash'
works fine :(

The error I get is 
Fatal error: Call to a member function performReadonlyTransformation() on a non-object in D:\wamp\www\mysite\framework\forms\FieldList.php on line 532

**Comment by nimeso** [comment]
As a side note... extending Member and using 

```php
public function updateMemberFormFields(FieldList $fields){
	$fields->removeByName("LastVisited");
}
```
works fine ... strange

**Comment by dhensby** [comment]
I think the extension works because it's running BEFORE the config setting is iterated over and then removed. This has been fixed by #4133
